harvard-huit/Harvard-Geospatial-Library
# Lock blacklight gemfile

**Lock blacklight gemfile**
* * *

# What does this Pull Request do?
This PR locks the Blacklight version in our Gemfile, fixing the issues we're having with changes upstream on that gem.

# How should this be tested?
Build this locally and see if it builds and comes up (just being able to bring it up was the issue we had). I have also deployed it to QA and it is running, but it can be deployed to dev if you would like to try on non-local environment.

# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests? N
- integration tests? N

# Interested parties
@enriquediaz
